spacer/torch_utils.py

```diff
@@ -0,0 +1,130 @@
+"""
+This file contains a set of pytorch utility functions
+"""
+
+import torch
+import numpy as np
+from spacer import models
+from torchvision import transforms
+from collections import OrderedDict
+
+
+def gray2rgb(im):
```

Comment:
move this to shared `extract_features` util?

Reply:
`gray2rgb()` moved to shared `extract_features_utils.py`